app/controllers/api/v1/todos_controller.rb

```diff
@@ -6,12 +6,12 @@ def index
   end
   def show
     @todo = Todo.find(params[:id])
```

Comment:
In Ruby add the `@` in front of a variable makes it an instance variable of the class. This is useful if the variable needs to be used by separate methods of the class or needs to be retained.

In general best practice would be to keep your scoping as small as possible:
Global < Class Level < Instance < Local (method)

In most cases in the class I believe you do not require an instance variable and instead could have the variable scoped locally to just the method. This will keep the code safer and ensure that variable isn't accidentally modified somewhere else in the code leading to bugs.